### **Chi-Square Test Assignment**

**Mizzare Corporation**
**Smart Home Devices & Customer Satisfaction**


**BACKGROUND**

Mizarre Corporation is the maker of two flagship products for the smart home: Smart Thermostats and Smart Lights. In order to grasp the user experience, the company carried out a customer satisfaction survey among 600 users.

Restaurant owners have voiced that satisfaction levels may vary between the two devices. In order to verify whether the type of device has a statistically significant effect on customer satisfaction, we perform the Chi-Square Test for Independence with a contingency table of observed frequencies.

**OBJECTIVE**

To find out whether there is a significant relationship between the device type (Smart Thermostat vs. Smart Light) and the customer satisfaction level by performing the Chi-Square test at α = 0.05.

**DATA PROVIDED**:

The data is summarized in a contingency table showing counts of customers in each satisfaction level for both device types:



| Satisfaction Level | Smart Thermostat | Smart Light | Total |
|--------------------|------------------|--------------|--------|
| Very Satisfied     | 50               | 70           | 120    |
| Satisfied          | 80               | 100          | 180    |
| Neutral            | 60               | 90           | 150    |
| Unsatisfied        | 30               | 50           | 80     |
| Very Unsatisfied   | 20               | 50           | 70     |
| **Total**          | **240**          | **360**      | **600**|














































## **Task 1: State the Hypotheses Statement**

**Null Hypothesis (H₀):**  
The type of device and customer satisfaction level are independent of each other.

H₀: The two variables are independent.


**Alternative Hypothesis (H₁):**  
Device type and satisfaction level are significantly related.
  
H₁: The two variables are dependent.


### **Task 2: Calculate the Test Statistic**

In [1]:
# Importing required libraries
import pandas as pd
from scipy import stats

# Creating observed frequency table
observed = pd.DataFrame({
    'Smart Thermostat': [50, 80, 60, 30, 20],
    'Smart Light': [70, 100, 90, 50, 50]
}, index=['Very Satisfied', 'Satisfied', 'Neutral', 'Unsatisfied', 'Very Unsatisfied'])

# Displayed observed data
print("Observed Frequencies:")
print(observed)

# Performing Chi-Square test
chi2, p, dof, expected = stats.chi2_contingency(observed)

# results
print(f"\nChi-Square Statistic: {chi2:.4f}")
print(f"P-value: {p:.4f}")
print(f"Degrees of Freedom: {dof}")

Observed Frequencies:
                  Smart Thermostat  Smart Light
Very Satisfied                  50           70
Satisfied                       80          100
Neutral                         60           90
Unsatisfied                     30           50
Very Unsatisfied                20           50

Chi-Square Statistic: 5.6382
P-value: 0.2278
Degrees of Freedom: 4


The test statistic is χ² = 5.6382, which we’ll compare with the critical value next.

**3: DETERMINE THE CRITICAL VALUE**:

**Significance level**: α = 0.05

**Degrees of Freedom**: (5 – 1) × (2 – 1) = 4

In [2]:
# Calculate critical value
alpha = 0.05
critical = stats.chi2.ppf(1 - alpha, dof)
print(f"Critical Value: {critical:.4f}")

Critical Value: 9.4877


**4: CHECKING IF THE RESULT SUPPORTS THE CLAIM**:

Now lets compare the calculated χ² value with the critical value.

In [3]:
# Decision rule
if chi2 > critical:
    print("Reject the null hypothesis.")
else:
    print("Fail to reject the null hypothesis.")

Fail to reject the null hypothesis.



**Since 5.6382 < 9.4877, we fail to reject the null hypothesis.
This means the sample data does not support the claim that device type affects customer satisfaction**

## **Conclusion:**

In accordance with the figures and Chi-Square test, the decision was made not to reject the null hypothesis.

It means that there is no statistical proof indicating that the kind of the device (Smart Thermostat vs. Smart Light) has a relationship with the level of customer satisfaction.

The differences in satisfaction that were observed could simply be attributable to random chance.